README.md

```diff
@@ -215,7 +215,7 @@ app.use exprestive
   * default: `__dirname`
 * `controllersPattern`
   * [Glob](https://github.com/isaacs/node-glob) pattern used to find controllers. Resolved relative to `appDir`
-  * default: `'controllers/*_controller.{coffee,js}'`
+  * default: `'controllers/*_controller.{coffee,js,ls}'`
```

Comment:
This is a default. We should enforce good behavior in our defaults. If someone really wants to do this they are free to override the `controllersPattern` setting